queen-application/src/main/java/fr/insee/queen/application/configuration/auth/AuthorityPrivileges.java

```diff
@@ -5,10 +5,9 @@ private AuthorityPrivileges() {
         throw new IllegalArgumentException("Constant class");
     }
 
-    public static final String HAS_USER_PRIVILEGES = "hasAnyRole('INTERVIEWER', 'REVIEWER', 'WEBCLIENT', 'ADMIN', 'REVIEWER_ALTERNATIVE', 'SURVEY_UNIT')";
-    public static final String HAS_INTERVIEWER_PRIVILEGES = "hasAnyRole('INTERVIEWER','ADMIN','WEBCLIENT')";
-    public static final String HAS_MANAGEMENT_PRIVILEGES = "hasAnyRole('INTERVIEWER', 'REVIEWER', 'WEBCLIENT', 'ADMIN', 'REVIEWER_ALTERNATIVE')";
+    public static final String HAS_USER_PRIVILEGES = "hasAnyRole('SURVEY_UNIT', 'REVIEWER', 'REVIEWER_ALTERNATIVE', 'INTERVIEWER', 'ADMIN', 'WEBCLIENT')";
+    public static final String HAS_SURVEY_UNIT_PRIVILEGES = "hasAnyRole('SURVEY_UNIT', 'INTERVIEWER', 'ADMIN', 'WEBCLIENT')";
+    public static final String HAS_INTERVIEWER_PRIVILEGES = "hasAnyRole('INTERVIEWER', 'ADMIN', 'WEBCLIENT')";
+    public static final String HAS_REVIEWER_PRIVILEGES = "hasAnyRole('REVIEWER', 'REVIEWER_ALTERNATIVE', 'INTERVIEWER', 'ADMIN', 'WEBCLIENT')";
```

Comment:
question: Interview in reviewer privileges?

Reply:
for now, the interviewer has the same privileges as the reviewer